src/cclib/parser/molproparser.py

```diff
@@ -222,18 +248,20 @@ def _parse_orbitals(self, inputfile, line):
                 line = next(inputfile)
             mocoeffs.append(coeffs)
 
-            # The loop should keep going until there is a double blank line, and there is
-            # a single line between each coefficient block.
+            # The loop should keep going until there is a double blank line,
+            # and there is # a single line between each coefficient block.
             line = next(inputfile)
             if not line.strip():
                 line = next(inputfile)
 
-        # If symmetry was used (offset was needed) then we will need to pad all MO vectors
+        # If symmetry was used (offset was needed)
+        # then we will need to pad all MO vectors
         # up to nbasis for all irreps before the last one.
         if offset > 0:
             for im, m in enumerate(mocoeffs):
                 if len(m) < self.nbasis:
-                    mocoeffs[im] = m + [0.0 for i in range(self.nbasis - len(m))]
+                    mocoeffs[im] = (
+                        m + [0.0 for i in range(self.nbasis - len(m))])
```

Comment:
I'm not sure this is correct. Doesn't this turn it into a list inside of a tuple? You should also never leave just one dangling delimiter. Either leave it as-is, or insert a newline after the last one as well. The original was easier to read.

Reply:
Commas not parentheses make a tuple.